sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-cosmos-multi-database-single-account/src/main/java/com/azure/spring/sample/cosmos/multi/database/DatasourceConfiguration.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.sample.cosmos.multi.database;
+
+import com.azure.cosmos.CosmosAsyncClient;
+import com.azure.cosmos.CosmosClientBuilder;
+import com.azure.spring.data.cosmos.config.AbstractCosmosConfiguration;
+import com.azure.spring.data.cosmos.config.CosmosConfig;
+import com.azure.spring.data.cosmos.core.ReactiveCosmosTemplate;
+import com.azure.spring.data.cosmos.core.convert.MappingCosmosConverter;
+import com.azure.spring.data.cosmos.repository.config.EnableReactiveCosmosRepositories;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class DatasourceConfiguration {
+
+    private static final String DATABASE1 = "database1";
+    private static final String DATABASE2 = "database2";
+
+    @Bean
+    public CosmosProperties primary() {
+        return new CosmosProperties();
+    }
+
+    @Bean
+    public CosmosClientBuilder primaryClientBuilder(CosmosProperties primaryProperties) {
+        return new CosmosClientBuilder()
+            .key(primaryProperties.getKey())
+            .endpoint(primaryProperties.getUri());
+    }
+
+    @EnableReactiveCosmosRepositories(basePackages = "com.azure.spring.sample.cosmos.multi.database.repository",
+        reactiveCosmosTemplateRef = "database1Template")
+    public class Database1Configuration extends AbstractCosmosConfiguration {
+
+        @Bean
+        public ReactiveCosmosTemplate database1Template(CosmosAsyncClient cosmosAsyncClient,
+                                                              CosmosConfig cosmosConfig,
+                                                              MappingCosmosConverter mappingCosmosConverter) {
+            return new ReactiveCosmosTemplate(cosmosAsyncClient, DATABASE1, cosmosConfig, mappingCosmosConverter);
+        }
+
+        @Override
+        protected String getDatabaseName() {
+            return DATABASE1;
+        }
+    }
+
+    @EnableReactiveCosmosRepositories(basePackages = "com.azure.spring.sample.cosmos.multi.database.repository2",
```

Comment:
Will it cause problem because there is no package named `...database.repository2`?

Reply:
It's better to do like this to make scenario clear:
1. Package `...repository1` have `User1` and `User1Repository`.
1. Package `...repository2` have `User2` and `User2Repository`.

Reply:
nope, it works normally

Reply:
Please delete these codes:
![image](https://user-images.githubusercontent.com/13167207/110724856-7f9eaf80-8251-11eb-8f38-933320493a63.png)

Use your code instead.